helm-chart/metabase/templates/deployment.yaml

```diff
@@ -78,7 +78,7 @@ spec:
               configMapKeyRef:
                 name: {{ include "metabase.fullname" . }}-configs
                 key: MB_JETTY_PORT
-          {{- if and (ne .Values.config.database.type "h2") (.Values.secrets.database.host) (.Values.secrets.database.password) (.Values.secrets.database.username) (.Values.secrets.database.port) (.Values.secrets.database.dbName) }}
+          {{- if and (ne .Values.config.database.type "h2") (.Values.secrets.database.host) (or .Values.secrets.database.password .Values.secrets.existingSecret.existingSecretPasswordKey) (or .Values.secrets.database.username .Values.secrets.existingSecret.existingSecretUsernameKey) (.Values.secrets.database.port) (.Values.secrets.database.dbName) }}
```

Comment:
What about:

* `.Values.secrets.existingSecret.secretName`
* `.Values.secrets.existingSecret.passwordKey`
* `.Values.secrets.existingSecret.usernameKey`
* `.Values.secrets.existingSecret.connectionURIKey`

Reply:
solved in the latest commit